modules/perform-on-demand-backups-s3-api-compatible-services.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-s3-api-compatible-services.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="perform-on-demand-backups-s3-api-compatible-services_{context}"]
+= Performing on-demand backups on an S3 API compatible bucket
+
+Use the {product-title-short} portal to trigger manual backups of {product-title} to an S3 API compatible bucket.
+
+.Prerequisites
+* You must have already integrated {product-title} with an S3 API compatible bucket.
```

Comment:
```suggestion
* You have integrated {product-title-short} with an S3 API compatible bucket.
```

Reply:
A suggestion - elsewhere you refer to the S3 API compatible service, not bucket. I was interpreting this as the service was something that you configured the integration with, and then the bucket is the storage device that you actually backed up to. So I think here you might want to use "you have integrated RHACS with an S3 API compatible service"?

Reply:
Yeah `S3 API compatible service` makes more sense here. I'll change it.